docs/diagrams/StudentClassDiagram.puml

```diff
@@ -0,0 +1,24 @@
+@startuml
+!include style.puml
+skinparam arrowThickness 1.1
+skinparam arrowColor MODEL_COLOR
+skinparam classBackgroundColor MODEL_COLOR
+
+
+Class Student
+Class StudentName
+Class StudentPhone
+Class ParentName
+Class ParentPhone
+Class Progress
+Class LessonName
+
+Student *--> "1"StudentName
+Student *--> "0..1"StudentPhone
+Student *--> "0..1"ParentName
+Student *--> "0..1"ParentPhone
+Student *--> "0..10"Progress
```

Comment:
I'm not too sure about this, but should this be changed to `"1"ProgressList` instead? Technically, a `Student` consists of a `ProgressList` that in turn consists of up to 10 `Progress` objects, so maybe this relationship can be reflected here?

Reply:
I'm not sure about this - if it was simply an array / ArrayList of Progress, would it still be joined to Progress or ArrayList<Progress>? I'm not very sure too but will leave it for now

Reply:
I've since changed this to follow yours for consistency